tests/test_meshtastic_utils_disconnect.py

```
@@ -0,0 +1,545 @@
+            if loop.is_running():
+                with contextlib.suppress(RuntimeError, ConcurrentTimeoutError):
+                    asyncio.run_coroutine_threadsafe(asyncio.sleep(0), loop).result(
+                        timeout=1
+                    )
+            else:
+                with contextlib.suppress(RuntimeError):
+                    loop.run_until_complete(asyncio.sleep(0))
+
+    @patch("mmrelay.meshtastic_utils.logger")
+    @patch("mmrelay.meshtastic_utils.reconnect", new_callable=AsyncMock)
+    def test_on_lost_meshtastic_connection_normal(self, mock_reconnect, mock_logger):
```

Ruff (0.15.12)

[warning] Unused method argument: `mock_reconnect`

(ARG002)